tests/unit/CMakeLists.txt

```diff
@@ -1,51 +1,52 @@
-# Copyright (C) 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
```

Comment:
Maybe don't modify existing license header

Reply:
It's a new CMakeLists.txt from our internal repo, the original CMakeLists.txt was moved to `tests/unit/tid-allocator/CMakeLists.txt`.

Reply:
Got it, thanks